packages/components/src/dropdown-menu/index.js

```diff
@@ -59,13 +58,13 @@ function DropdownMenu( dropdownMenuProps ) {
 		noIcons,
 	} = dropdownMenuProps;
 
-	if ( isEmpty( controls ) && ! isFunction( children ) ) {
+	if ( ( ! controls || ! controls.length ) && ! isFunction( children ) ) {
 		return null;
 	}
 
 	// Normalize controls to nested array of objects (sets of controls)
 	let controlSets;
-	if ( ! isEmpty( controls ) ) {
+	if ( controls && controls.length ) {
```

Comment:
same here

Reply:
Addressed as part of 242097f994ba0b0dc6d989f35d6b90a8b3f96f37.